web/src/DetailedInfoPage.tsx

```diff
@@ -372,6 +391,133 @@ export const DetailedInfoPage: React.FunctionComponent<{}> = () => {
         });
     };
 
+    const renderSeqSection = () => {
+        if (!sample || !sequenceInfo) {
+            return <></>;
+        }
+        return (
+            <>
+                <h4
+                    style={{
+                        borderBottom: `1px solid black`,
+                    }}
+                >
+                    Sequence Information
+                </h4>
+                {renderSeqInfo(sequenceInfo)}
+                {prepReadMetadata(sequenceInfo.meta || {})}
+            </>
+        );
+    };
+
+    const renderSampleSection = () => {
+        if (!sample || !sampleInfo) {
+            return <></>;
+        }
+        return (
+            <>
+                <h4
+                    style={{
+                        borderBottom: `1px solid black`,
+                    }}
+                >
+                    Sample Information
+                </h4>
+                {Object.entries(sampleInfo)
+                    .filter(([key, value]) =>
+                        sampleFieldsToDisplay.includes(key)
+                    )
+                    .map(([key, value]) =>
+                        value ? (
```

Comment:
I think we should display keys with falsey, especially this will probably catch falsey like `false, 0, etc`.

Reply:
Good point - this was meant to catch instances where a field is null (or undefined), but false or 0 are both valid. 
Replaced with `{value != null ? value.toString() : "null/undefined"}`